media/css/pebbles/components/_footer.scss

```diff
@@ -5,33 +5,61 @@
 @import '../includes/lib';
 
 #colophon {
+    @include open-sans;
     @include font-size(14px);
-    background: #fff;
-    color: $color-text-primary;
+    background: #000;
+    color: #fff;
     line-height: 1.5;
```

Comment:
Is this line height meant to remain at 1.5? It is consistent with the rest of the site, but it makes the footer pretty tall... Not a blocker - just checking.